pennylane_lightning/core/src/simulators/lightning_qubit/gates/cpu_kernels/GateImplementationsLM.hpp

```diff
@@ -1806,10 +1696,10 @@ class GateImplementationsLM : public PauliGenerator<GateImplementationsLM> {
         const PrecisionT sj =
             inverse ? -std::sin(angle / 2) : std::sin(angle / 2);
         auto core_function =
-            [cr,
-             sj](std::complex<PrecisionT> *arr, const std::size_t i0011,
-                 const std::size_t i1100,
-                 [[maybe_unused]] const std::array<std::size_t, 16> &indices) {
+            [cr, sj](std::complex<PrecisionT> *arr, const std::size_t i0011,
```

Comment:
Why not passing them by reference? :thinking: 
```suggestion
            [&cr, &sj](std::complex<PrecisionT> *arr, const std::size_t i0011,
```

Reply:
`PrecisionT` can be float or double, so the variables are POD.